protected/humhub/modules/comment/widgets/views/comment.php

```diff
@@ -91,6 +92,8 @@
         <div class="wall-entry-controls">
             <?php if ($module->canComment($comment)): ?>
                 <?= CommentLink::widget(['object' => $comment]); ?>&nbsp;&nbsp;&middot;&nbsp;
+            <?php elseif (Comment::isSubComment($comment) && $module->canComment($comment->content->getPolymorphicRelation())): ?>
```

Comment:
Maybe it make sense to move this into the `CommentLink` widget.

Reply:
Done!